### DISCUSSION TITLE
laravel: file autocomplete

### DISCUSSION THREAD

**Discussion by mdurao** [ohmyzsh/ohmyzsh#11946] [Help]
Having this:

```
# Laravel5 basic command completion
_laravel5_get_command_list () {
	php artisan --raw --no-ansi list | sed "s/[[:space:]].*//g"
}

_laravel5 () {
  if [ -f artisan ]; then
    compadd `_laravel5_get_command_list`
  fi
}

compdef _laravel5 artisan

#Alias
alias art='php artisan'
```

How can I have FILE autocompletion after the laravel commands?

`art<TAB>_laravel5<TAB>_FILES`

Can someone help me or lead me to the right direction? Thank you.

**Comment by mcornella** [comment]
You need to run `compdef _laravel5 art` to enable completion with the `art` alias.

**Comment by mdurao** [comment]
Hi, @mcornella 

I have completion with the command and the alias, what I'm not able to do is having filesystem autocompletion after a laravel command.

Example:

`php artisan import:csv ~/` <-- after writing this, if you press `<TAB>` it will cycle again through artisan commands instead of cycling through the files in the current dir!
